.changelog/1794.trivial.md

```diff
@@ -0,0 +1 @@
+When auto-redirecting from landing page, preserve scope
```

Comment:
I would change "preserve" to "store"
("preserve" to me implies it isn't removed during a redirect. "store" implies longer time storage)

Reply:
Updated to:

> When auto-redirecting from landing page, restore stored scope